homeassistant/components/icloud/strings.json

```
@@ -26,13 +26,13 @@
       }
     },
     "error": {
-      "login": "Login error: please check your email & password",
+      "login": "[%key:common::config_flow::error::invalid_auth%]",
```

Please change the key as well and adjust config_flow and the tests accordingly:
```suggestion
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
```